packages/frontend-api/src/Resources/config/graphql-types/ProductFilter/ProductFilterDecorator.types.yaml

```diff
@@ -0,0 +1,24 @@
+ProductFilterDecorator:
+    type: input-object
+    decorator: true
+    config:
+        description: "Represents a product filter"
+        fields:
+            minimalPrice:
+                type: "Money"
+                description: "Minimal price filter"
+            maximalPrice:
+                type: "Money"
+                description: "Maximal price filter"
+            brands:
+                type: "[Uuid!]"
+                description: "Array of uuids of brands filter"
+            flags:
+                type: "[Uuid!]"
+                description: "Array of uuids of flags filter"
+            inStock:
```

Comment:
While testing I found out this field does not do anything. 
I get the same results while this was set to `null`, `true`, or `false`. (some filters – brand, flag – were applied too).

Can you please verify this?

Reply:
I'm not sure if the fix is enough. 

`inStock: true` – returns only products in stock
`inStock: false` and `inStock: null` (or without this at all) returns products in stock and out of stock. 

I don't know if it should behave differently when I set false (only products out of stock). 
I know it's not currently implemented on frontend.

Maybe @sslt can join this thought?

Reply:
Well, there is hardly scenario, when somebody needs only list of products not in stock. So... how about we rename the atribute from "inStock" to "onlyInStock". If the value provided is true, we return only products in stock, in any other case we return all products. Same as current frontend.

Reply:
de428a76000186cc990deb261bcbfe8f3f420dae